### PR TITLE
Added using-marquee.py

### PR DESCRIPTION
**Pull request by VishalAmbavade**

### CODE REVIEW

**Comment by Goku-kun** [comment]
Brilliantly coded @VishalAmbavade . The execution along with the exception handling with neat code is a job well done. I'll consider this merge request. And since you wrote this code, I'd ask you to make a small change in the code and submit a new pull request after this one. The change is to remove the input method to get the string to be printed and replace it with a permanent "Hello, World!" string since that is the goal of this repository, right?

**Comment by VishalAmbavade** [comment]
Hi @Goku-kun, can you please review my PR?

File: using-marquee.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import sys
+import time
+
+def main():
+    program = animate()
+    program.data = input("Enter string : ") + " "
```
Comment (Goku-kun) [inline using-marquee.py:6]: Change the data with a permanent string "Hello, World!" since the sole purpose of this repository is to print "Hello, World!"

**Review by Goku-kun — COMMENTED** [review]
There is only one change that is to be made in the file which I've mentioned while reviewing the file.

**Pull request merged**